src/components/reusable/contactLink/ContactLink.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {FC, JSX} from "react";
+import './contact-link.scss';
+import SvgSprite from "../svgSprite/SvgSprite";
+
+interface TProps {
+    srcType: 'link' | 'phone' | 'email',
+    title: string,
+    src: string,
+    iconId: string,
+    target?: string
```

Comment:
you should create another type, which will extend this one and will have target as a required type. from TProps it must be removed. It is a bad practice to add million of optional params into same interface

Reply:
I don't understand what's meant here. I don't know how to implement it